modules/apple_support/0.11.0/presubmit.yml

```diff
@@ -0,0 +1,4 @@
+platforms:
+  macos:
```

Comment:
I understand apple_support only makes sense for builds running on macOS. But projects depend on apple rules may still need to build on other platforms (eg. TensorFlow). Therefore, should we at least make sure the targets we expose can be loaded correctly on other platforms? (eg. `bazel build --nobuild @apple_support//lib/...` works on Windows and Linux) This will help ensuring bazel query doesn't break when inspecting this dependency on other platforms.

Reply:
This is a good point and makes completely sense, added.

Reply:
We recently added[ matrix testing ](https://github.com/bazelbuild/continuous-integration/pull/1300)for Bazel CI. Can you use matrix testing to simplify the presubmit.yml. I'll later update the add_module.py tool to generate presubmit.yml in this new style.